fix: redeploying store value changes (#875)
appsmithorg/appsmith-docs: website/docs/reference/appsmith-framework/widget-actions/store-value.md

---
# Store Value

> The storeValue() function saves data in the browser as key-value pairs and can be later accessed anywhere in the application. Appsmith’s storeValue() works similarly to the browser's localStorage method.

Reviewer: ```suggestion
The storeValue() function saves data in the browser as key-value pairs and can be later accessed anywhere in the application.
```

| Argument Name | Description  |
| ------------- |--------------------------- |
| **key**       | Name of the key.            |
| **value**     | The data you want to save using storeValue(). You can save any data type in the Appsmith store.  |
| **persist**   | Defaults to **true**. **True** persists the key-value pair in the browser's local storage, and you can use it between sessions. A **false** doesn't persist the value and removes the key once the page refreshes or is closed. |

**Example 1:** if you want to store the text of an input widget, you can use the storeValue() as shown below:

```javascript

> {{storeValue('email',email1.text)}}

Reviewer: ```suggestion
{{storeValue('email',Input1.text)}}
```

```

> Here, `email` is the key where the value is stored and `email1.text` is the value in the input widget that's saved in the storage object.

Reviewer: ```suggestion
Here, `email` is the key where the value is stored and `Input1.text` is the value in the input widget that's saved in the Appsmith store.
```

{{ appsmith.store.key }}
```

> **Example**: in the example above, you stored the value of `email1.text`. You can access this value anywhere in the application by referencing the key `email`:

Reviewer: ```suggestion
**Example**: In the example above, you stored the value of `Input1.text`. You can access this value anywhere in the application by referencing the key `email`:
```
